site/app/libraries/PollUtils.php

```diff
@@ -26,6 +26,7 @@ public static function getPollExportData(array $polls): array {
                 "id" => $poll->getId(),
                 "name" => $poll->getName(),
                 "question" => $poll->getQuestion(),
+                "question-type" => $poll->getQuestionType(),
```

Comment:
I think it would be best to replace the - in question-type to an  underscore to follow the naming convention.

Reply:
You're right. I'll change that